packages/playwright-core/src/server/frames.ts

```diff
@@ -507,46 +507,41 @@ export class Frame extends SdkObject {
       callback(error);
   }
 
-  async nonStallingRawEvaluateInExistingMainContext(expression: string): Promise<any> {
+  async nonStallingEvaluate<T>(cb: () => Promise<T>): Promise<T> {
```

Comment:
This is no longer a  non-stalling evaluate, it is now `raceAgainstFrameNavigation` ?